src/resources/ha-style.js

```diff
@@ -188,12 +188,19 @@ documentContainer.innerHTML = `<custom-style>
         color: var(--primary-color);
         font-weight: 500;
       }
+      .card-actions ha-call-ws-api-button:not([disabled]) {
+        color: var(--primary-color);
+        font-weight: 500;
+      }
 
       .card-actions paper-button.warning:not([disabled]),
       .card-actions ha-call-api-button.warning:not([disabled]),
       .card-actions ha-call-service-button.warning:not([disabled]) {
         color: var(--google-red-500);
       }
+      .card-actions ha-call-ws-api-button.warning:not([disabled]) {
```

Comment:
Please add this declaration to the previous one.

Reply:
Not sure I follow this comment... also, does this still apply with the other button comment?

Reply:
No need for a second style that does the same thing
```suggestion
      .card-actions paper-button.warning:not([disabled]),
      .card-actions ha-call-api-button.warning:not([disabled]),
      .card-actions ha-call-service-button.warning:not([disabled]),
      .card-actions ha-call-ws-api-button.warning:not([disabled]) {
        color: var(--google-red-500);
      }
```

Reply:
Awesome, that makes sense now. Thx